lib/tag-finder.coffee

```diff
@@ -8,14 +8,14 @@ SelfClosingTags = require './self-closing-tags'
 module.exports =
 class TagFinder
   constructor: (@editor) ->
-    @tagPattern = /(<(\/?))([^\s>]+)([\s>]|$)/
-    @wordRegex = /[^>\r\n]*/
+    @tagPattern = /(<(\/?))([^/\s>]+)([^/>]*)(\/?)(>)/
```

Comment:
The regex `(<(\/?))([^/\s>]+)([^/>]*)(\/?)(>)` won't match `<foo bar="/">`. This is because `[^/>]` doesn't mean "not the sequence `/>`", but match all characters with the exception of `/` and `>`. Same goes for the regex in `patternForTagName`. Maybe `(<(\/?))([^/\s>]+)((?:[^/>]|[^>](?!>))*)(\/?)(>)` could be used instead. In this regex the pattern `[^/>]*` is replaces by `((?:[^/>]|[^>](?!>))*`, which will again match all characters with the exception of `/` and `>`, but also the character `/` if it isn't followed by the character `>`.